make/os_win

```diff
@@ -17,7 +17,11 @@ EXE = .exe
 O_STANC = 3
 CFLAGS_GTEST += -DGTEST_HAS_PTHREAD=0
 PATH_SEPARATOR = '\'
-BIT = 64
+ifeq ($(PROCESSOR_ARCHITECTURE)$(PROCESSOR_ARCHITECTURE6432),x86)
```

Comment:
I'm confused with the two variables. Are both needed? It looks like the second one is misspelled (`PROCESSOR_ARCHITECTURE6432`).
I'm looking at this as a reference:
http://blogs.msdn.com/b/david.wang/archive/2006/03/26/howto-detect-process-bitness.aspx

Will this set the `BIT` variable for an AMD processor?